### PR TITLE
Separated out the project dependencies from build.sbt to Dependencies…

### PR DESCRIPTION
**Pull request by dagaharshit**
This commit will take out all the external library dependencies into a new file such that it would be easier to maintain all the external dependencies.

### CODE REVIEW

**Comment by adamw** [comment]
Thanks! Though I must admit I'm not a fan of such a separation - for example, you can't see what are the dependencies for specific modules. But maybe other might weight in with their comments?

**Comment by annabee** [comment]
I prefer having dependencies in a separate file too (much easier to read), but I would separate them logically into collections corresponding to modules.

File: project/Dependencies.scala [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  val betterFiles = "com.github.pathikrit" %% "better-files" % betterFilesVersion
+
+  val logbackClassic = "ch.qos.logback" % "logback-classic" % logbackClassicVersion
+
+  /**
+    * Libraries to be used in Test.........
+    */
+
+  val braveInstrumentationHttpTest = "io.zipkin.brave" % "brave-instrumentation-http-tests" % braveVersion
```
Comment (annabee) [inline project/Dependencies.scala]: I would suggest using `% test` when declaring the dependencies. I think it makes more sense.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import sbt._
+import Keys._
+
+object Dependencies {
+
+  val scalaVer = "2.12.6"
```
Comment (annabee) [inline project/Dependencies.scala]: What do you think about factoring the version numbers into a separate Version object?

File: project/Dependencies.scala (hunk 1 above)
Comment (dagaharshit) [reply project/Dependencies.scala]: I've updated them.

File: project/Dependencies.scala (hunk 2 above)
Comment (dagaharshit) [reply project/Dependencies.scala]: Moved to Versions object in same file.

**Comment by dagaharshit** [comment]
Hi @annabee. I have made the required changes. Requesting you to kindly review. Thanks !

**Comment by darkfrog26** [comment]
I also prefer dependencies to be in the `build.sbt` file as it makes it a particular pain for people just taking a look at the project for the first time trying to decide what dependencies it has.

**Comment by adamw** [comment]
Closing for now due to lack of feedback.

**Pull request closed without merging**